.github/workflows/build-release.yml

```
@@ -19,6 +19,8 @@ on:
 env:
   PROJECT_PATH: ./Source/HedgeModManager.UI/HedgeModManager.UI.csproj
   FLATPAK_ID: io.github.hedge_dev.hedgemodmanager
+  MACOS_ID: com.hedge_dev.hedgemodmanager
```

Change package id to be consistent with flatpak.
```suggestion
  MACOS_ID: io.github.hedge_dev.hedgemodmanager
```